rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -100,8 +101,8 @@ def persist(self, model_dir):
         # type: (Text) -> Dict[Text, Any]
         file_name = self.name+".json"
         full_name = os.path.join(model_dir, file_name)
-        with io.open(full_name, 'w') as f:
-            f.write(json.dumps({"dimensions": self.dimensions}))
+        with io.open(full_name, 'w', encoding="utf-8") as f:
```

Comment:
since i specify "duckling_dimensions" : ["time"] in the config, so when training, self.dimensions already contains a <list> of unicode strings [u'time']... so available_dimensions is never used in the creation of the json file that causes the error (unless i don't specify anything in the config file obviously)

i tried wrapping with str() and with that the error now says "TypeError: must be unicode, not str"
so I'm either doing something wrong (I'm not the best python person)

being the string is already unicode I'm not sure what to do except for the fix i already did:)